config/m4/ucm.m4

```diff
@@ -48,3 +48,8 @@ esac
 AM_CONDITIONAL([HAVE_UCM_PTMALLOC283],[test "x$HAVE_UCM_PTMALLOC283" = "xyes"])
 AM_CONDITIONAL([HAVE_UCM_PTMALLOC286],[test "x$HAVE_UCM_PTMALLOC286" = "xyes"])
 
+AC_CHECK_FUNCS([malloc_get_state malloc_set_state],
+               [],
+               [],
+               [#include <stdlib.h>])
```

Comment:
shouldn't this be `#include <malloc.h>`?

Reply:
this is fix for GCC compilation warning:
```
ucs/test_log.cc:14:18: error: ‘%s’ directive output may be truncated writing up to 63 bytes into a region of size 59 [-Werror=format-truncation=]
     virtual void init() {
                  ^~~~
ucs/test_log.cc:29:17: note: ‘snprintf’ output between 6 and 69 bytes into a destination of size 64
         snprintf(ucs_log_spec, sizeof(ucs_log_spec), "file:%s", logfile);
         ~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
according man pager stdlib.h should be used